cmd/sand-agent-cli/network.go

```diff
@@ -61,12 +61,28 @@ func (a *App) NetworksList(c *cli.Context) error {
 }
 
 func (a *App) NetworkDelete(c *cli.Context) error {
+	ctx := context.Background()
 	client, err := a.sandClient(c)
 	if err != nil {
 		return err
 	}
 
-	err = client.NetworkDelete(context.Background(), c.String("network"))
+	if c.Bool("recursive") {
+		endpoints, err := client.EndpointsList(ctx, params.EndpointsList{
+			NetworkID: c.String("network"),
+		})
+		if err != nil {
+			return err
+		}
+		for _, endpoint := range endpoints {
+			err := client.EndpointDelete(ctx, endpoint.ID)
```

Comment:
Can you add an output each time an endpoint is deleted? To let the user know what's happening?

Reply:
done